common/source/docs/common-mambaf405-mini.rst

```diff
@@ -0,0 +1,161 @@
+
+- CRSF also requires a TX1 connection, in addition to RX1, and automatically provides telemetry. Set :ref:`SERIAL1_OPTIONS<SERIAL7_OPTIONS>` to "0".
+
+- SRXL2 requires a connection to TX1 and automatically provides telemetry.  Set :ref:`SERIAL1_OPTIONS<SERIAL1_OPTIONS>` to "4".
+
+Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
+
+FrSky Telemetry
+===============
+
+FrSky Telemetry is supported using any UART TX pin. You need to set the following parameters to enable support for FrSky S.PORT (example given for UART4)
+
+- :ref:`SERIAL4_PROTOCOL<SERIAL4_PROTOCOL>` = 10
+- :ref:`SERIAL4_OPTIONS<SERIAL4_OPTIONS>` = 7
+
+OSD Support
+============
+
+The Mamba MK4 F405Mini supports internal OSD using :ref:`OSD_TYPE<OSD_TYPE>` = 1 (MAX7456 driver).
+
+DJI Video
+=========
+
+A standard DJI video connector is provided. Set :ref:`OSD_TYPE<OSD_TYPE>` to "3" (MSP) or "5" (DisplayPort) depending on use, and set :ref:`SERIAL3_PROTOCOL<SERIAL3_PROTOCOL>` to "32" (MSP) or "42" (DisplayPort) also.
+
+PWM Output
+===========
+
+The Mamba MK4 F405Mini supports up to 7 PWM outputs, all are DShot capable. The pads for motor output M1 to M4 are provided on  the motor connector and are BDShot capable, plus there is M7 on a separate pad for LED strip or another PWM output.
+
+The PWM is in 4 groups:
+
+-   PWM 1,2 in group1  (if used for :ref:`GPIOs <common-gpios>` their reference pin numbers are 50 and 51, respectively)
+-   PWM 3,4 in group2 (if used for :ref:`GPIOs <common-gpios>` their reference pin numbers are 52 and 53, respectively)
+-   PWM 5,6 in group3 (if used for :ref:`GPIOs <common-gpios>` their reference pin numbers are 54 and 55, respectively)
+-   PWM 7 (LED) in group4 (if used for :ref:`GPIOs <common-gpios>` its reference pin number is 56)
+
+Channels within the same group need to use the same output rate. If any channel in a group uses DShot then all channels in the group need to use DShot. Channels 1-4 support bi-directional DShot.
+
+Battery Monitoring
+==================
+
+The board does not have a built-in current sensor. The voltage sensor can handle up to 6S LiPo batteries.
+
+Typical battery setting parameters are:
+
+-   :ref:`BATT_MONITOR<BATT_MONITOR>` = 4
+-   :ref:`BATT_VOLT_PIN<BATT_VOLT_PIN>` = 11
+-   :ref:`BATT_CURR_PIN<BATT_CURR_PIN>` = 13
+-   :ref:`BATT_VOLT_MULT<BATT_VOLT_MULT>` = 11.1
+-   :ref:`BATT_AMP_PERVLT<BATT_AMP_PERVLT>` = 64 (will depend on external current sensor)
+
+Compass
+=======
+
+The Mamba MK4 F405Mini does not have a builtin compass, but you can attach an external compass using I2C on the SDA and SCL pads.
+
+RSSI
+====
+
+Analog RSSI input, if used, :ref:`RSSI_ANA_PIN<RSSI_ANA_PIN>` should be set to "12".
+
+Firmware
+========
+
+Firmware for this board can be found `here <https://firmware.ardupilot.org>`_ in  sub-folders labeled "Mamba MK4 F405Mini"".
```

Comment:
Directory doesn't exist - think you mean MambaF405-2022

Reply:
again a global replace messup...will fix...thanks